### PR TITLE
created requirements file

### PR DESCRIPTION
**Pull request by tiagokretzerwalter**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+python==3.9.13
+django==4.0
+djangorestframework
```
Comment (gabrielpalves) [inline requirements.txt:4]: Why is there not a version?

**Review by brunosphair — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+python==3.9.13
```
Comment (brunosphair) [inline requirements.txt:2]: Why using python 3.9.13 instead of the latest version?

**Review by matheusmachadomtt — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (tiagokretzerwalter) [reply requirements.txt:4]: Because we can use the latest version

File: requirements.txt (hunk 2 above)
Comment (tiagokretzerwalter) [reply requirements.txt:2]: Because Azure Functions only works with Python 3.9

Comment (brunosphair) [reply requirements.txt:2]: confirmed: [link](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=asgi%2Capplication-level)

Comment (tiagokretzerwalter) [reply requirements.txt:2]: [teste]([https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=asgi%2Capplication-level](https://github.com/tiagokretzerwalter/wedding-planner/pull/url))

File: requirements.txt (hunk 1 above)
Comment (gabrielpalves) [reply requirements.txt:4]: I checked in the [Django website](https://www.django-rest-framework.org/#requirements)

File: requirements.txt (hunk 2 above)
Comment (tiagokretzerwalter) [reply requirements.txt:2]: [Correct](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python?tabs=asgi%2Capplication-level)

**Review by gabrielpalves — APPROVED** [review]

**Pull request merged**